### ISSUE TITLE
RotaryCraft Support

### ISSUE THREAD

**Issue by covers1624** [covers1624/PowerConverters#6]
^^

**Comment by RoloEdits** [comment]
This Would Be Amazing

**Comment by walkerjonny** [comment]
Depends on Reika's mood to this now.
portablejim wanted to do this , too but was not allowed by Reika as I remember...
But that was at least before about six month in the past.
So, maybe, Raika has changed his mood on this now...

**Comment by covers1624** [comment]
Oh well, ill add it anyway, if Reika has an issue with it he can talk to me about it.

**Comment by RoloEdits** [comment]
Ok, I don't want him to get mad at you though.He has good intentions about balance and progression in his mods and they all make really good sense,but after a while I feel like the early, mid game is a little to constrictive. So My intentions would be just to take the edge off the the constrictive nature of the mid game.

**Comment by covers1624** [comment]
That is true, i could possibly keep in mind his progression and find some sort of mid point for resources that are needed to craft the components.

**Comment by RoloEdits** [comment]
I should fore warn i'm am horrible with balance,but perhaps having the recipe involve spring steel as that is somewhat mid game. and perhaps tiers? changing out the spring steel for inductive ingots and then finally a bedrock alloy ingot for maybe an uncapped version. An uncapped version is probably a little OP but then we aren't forced to build a reactor to use rf as a means of transporting energy. but make it unable to be used with a reactor as there is already a multi-block for that and would unfairly bypass that.

**Comment by covers1624** [comment]
Ill take that in to consideration, ill most likely just bounce ideas off reika to make it balanced and come to an agreement on it.

**Comment by RoloEdits** [comment]
i think that would be best, he's very smart when it comes to thinking how something would affect progression and balance before it's even implemented.

**Comment by covers1624** [comment]
i have to finish some stuff off for another project then finish the new GUI for PC then ill contact reika on irc and see what he has to say.

**Comment by RoloEdits** [comment]
there is no rush at all. please, finish what ever you have to do before this.

**Comment by covers1624** [comment]
dont worry, i will get arround to it eventually if MattaBase dosent keep throwing me projects.
